src/main/java/io/mosip/digitalcard/dto/DataShareDto.java

```diff
@@ -0,0 +1,36 @@
+package io.mosip.digitalcard.dto;
+
+import lombok.Data;
+
+
+
+/**
+ * Instantiates a new data share.
+ */
+
+/**
+ * Instantiates a new data share.
+ */
+@Data

```

Comment:
Please remove the above duplicate comments

Reply:
okay

Reply:
I can see that is not yet resolved

Reply:
taken care